src/Workspaces/Core/MSBuild.BuildHost/Microsoft.CodeAnalysis.Workspaces.MSBuild.BuildHost.csproj

```diff
@@ -39,6 +51,11 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="..\..\..\Compilers\Core\Portable\InternalUtilities\*.cs" Link="InternalUtilities\%(FileName).cs" />
+
+    <!-- Exclude utilities that use System.Reflection.Metadata -->
+    <Compile Remove="..\..\..\Compilers\Core\Portable\InternalUtilities\BlobBuildingStream.cs" />
+    <Compile Remove="..\..\..\Compilers\Core\Portable\InternalUtilities\IncrementalHashExtensions.cs" />
```

Comment:
Why did we need to do these removes?

Reply:
We would need to add reference to SRM, which is otherwise not needed. These helpers would bring it in but they are not used by the build host.

Reply:
It would be OK to `PackageDownloadAndReference`  the SRM provided by MSBuild, but no need to do so if you don't actually need the dependency.